docs/operator-manual/user-management/index.md

```diff
@@ -196,6 +197,93 @@ NOTES:
   Argo CD will automatically use the correct `redirectURI` for any OAuth2 connectors, to match the
   correct external callback URL (e.g. `https://argocd.example.com/api/dex/callback`)
 
+## OIDC Configuration with DEX
```

Comment:
I wasn't sure if this should go under this "Dex" section or if it should be under "Existing OIDC Provider" but I stuck it with "Dex" to group things together.